deps.sh

```diff
@@ -136,7 +136,7 @@ check_fedora_pkgs () {
 }
 
 check_debian_pkgs () {
-  local REQUIRED_DEBS=( perl autoconf gettext automake autopoint flex bison build-essential gcc-multilib protobuf-compiler llvm lcov )
+  local REQUIRED_DEBS=( perl autoconf gettext automake autopoint flex bison clang build-essential gcc-multilib protobuf-compiler llvm lcov )
```

Comment:
This should probably be `libclang` ?

Reply:
Hmmm, I personally don't think that would be best considering [fedora](https://github.com/firedancer-io/firedancer/blob/main/deps.sh#L115C83-L115C83) is also installing `clang` vs `libclang`. It's obviously up to you, and I'll defer to what you what you'd prefer. But I was trying to some form of parity and `libclang` is not a dependency of any distro currently in this file

Reply:
I don't think we need full `clang` and that one is probably wrong too, but this is fine for now.